is.json

```diff
@@ -0,0 +1,864 @@
+        "World.Category.MyWorlds" : "Veraldarnar mínar",
+        "World.Category.Custom" : "Sérsnið",
+
+        "World.AccessLevel.Anyone" : "Hver sem er",
+        "World.AccessLevel.RegisteredUsers" : "Skráðar notendur",
+        "World.AccessLevel.Friends" : "Tengiliðir",
+        "World.AccessLevel.LAN" : "LAN",
+        "World.AccessLevel.Private" : "Einkaþing (einungis boð)",
+
+        "World.SortParameter.SearchScore" : "Search Score",
+        "World.SortParameter.Name" : "Nafn",
+        "World.SortParameter.OpenedWorldCount" : "Opnaðar veraldar",
+        "World.SortParameter.TotalSessionCount" : "Öll þing",
+        "World.SortParameter.TotalUserCount" : "Heildarnotendur",
+        "World.SortParameter.CreationDate" : "Stofndagur",
+        "World.SortParameter.LastUpdateDate" : "Síðast breytt",
+        "World.SortParameter.FirstPublishTime" : "Útgefnardagur",
+        "World.SortParameter.TotalVisits" : "Heildarheimsóknir",
+
+        "World.Filter.DataSources" : "Gagnaheimildir:",
+        "World.Filter.OpenedWorlds" : "Opnaðar veraldar",
+        "World.Filter.ActiveSessions" : "Virk þing",
+        "World.Filter.PublishedWorlds" : "Útgefnar veraldar",
+        "World.Filter.LocalWorlds" : "Heimagistaðar veraldar",
+        "World.Filter.BaseSearch" : "Grunnleit:",
+        "World.Filter.ByOwner" : "Af eiganda:",
+        "World.Filter.UserOwner" : "Eigandi er notandi",
+        "World.Filter.GroupOwner" : "Eigandi er hópur",
+        "World.Filter.SubmittedToGroup" : "Lögð í hóp:",
+        "World.Filter.Featured" : "Einungis það valin sérstaklega",
+        "World.Filter.MyWorlds" : "Heimarnar mínar",
+        "World.Filter.SessionFiltering" : "þingsía:",
+        "World.Filter.MinimumUsers" : "Lámarksnotendur: <b>{n}</b>",
+        "World.Filter.MinimumFriends" : "Lámarkstengiliðir: <b>{n}</b>",
+        "World.Filter.OnlyHeadless" : "Einungis höfuðlausar hýsendur",
+        "World.Filter.MaxAccessLevel" : "Hámarks aðgangstig:",
+        "World.Filter.MinimumUptime" : "Lámarks upptími:",
+        "World.Filter.Sorting" : "Flokkun:",
+
+        "Tools.AvatarCreator" : "Avatar smíðara",
+        "Tools.FullBodyCalibrator" : "Full Body stillingagræja",
+        "Tools.CameraStreaming" : "Myndavél / Streymara",
+        "Tools.CreateNewWorld" : "Búa til nýja veröld",
+        "Tools.DepositNCR" : "NCR innborgun",
+        "Tools.WithdrawNCR" : "NCR útborgun",
+        "Tools.Debug" : "Debug",
+
+        "Options.FreeformDash.On" : "Fljótandi mælaborð: Á",
+        "Options.FreeformDash.Off" : "Fljótandi mælaborð: Af",
+        "Options.SeatedMode.On" : "Sitjandakerfi: Á",
+        "Options.SeatedMode.Off" : "sitjandakerfi: Af",
+
+        "Indicator.OnlineUsers" : "Online notendur: ~{num_total} ({num_registered})",
+        "Indicator.ServerStatus" : "<color=#fff>Netþjónusta:</color> {status, select, Gótt {Good} Hægt {Slow} Niður {Down} Ekkert netsamband {No Internet}} ({response_ms} ms)",
```

Comment:
For the select formatter, the translated string is inside of the brackets. So Gótt {Good} would be Good {Gótt}